### ISSUE TITLE
Environment no longer has an impact on blocks

### ISSUE THREAD

**Issue by Suim67** [JimiIT92/UniverseGuard2#182]
Hi!
![image](https://user-images.githubusercontent.com/80638458/114781252-bbf58c00-9d78-11eb-8f6c-503f23362a61.png)
![image](https://user-images.githubusercontent.com/80638458/114781313-cd3e9880-9d78-11eb-9dcd-d39e2df9fda4.png)

In 2.35 version of Universeguard, as you can see, blocks seems to be stuck in the state that they've when they are set
I've send you the flags of my region, maybe i am doing something wrong ?

**Comment by JimiIT92** [comment]
Is it referring to the grass blocks not turning into dirt, right? If that is the case the cause is the place flag: this flag also prevents blocks like grass or mycelium from spreading. In the same way it prevents grass from turning into dirt. I might add 2 separate flags for this events, so you can handle them separately

**Comment by Suim67** [comment]
Yes for your question, but also for the fluids, when place flag is "false", fluids are not moving.
Its a good idea yes !
